### PR TITLE
add webinar link to host activation email

### PR DESCRIPTION
**Pull request by alanna**

### CODE REVIEW

File: templates/emails/activated.collective.as.host.hbs [hunk 1]
```diff
@@ -23,9 +23,11 @@ Contributions made to your Collectives via credit card (through Stripe) or or ot
   <li>How to <a href="https://docs.opencollective.com/help/fiscal-hosts/fiscal-host-fees">set your Host Fee</a> (what you charge Collectives)</li>
 </ul>
 
-<p>I hold monthly open Zoom calls for Fiscal Host admins, a space for asking questions and learning from other Hosts. Let me know if you want an invite! You're also welcome to <a href="https://slack.opencollective.com">join our Slack</a> and jump into the #hosts channel.
+<p>I hold <a href="https://events.humanitix.com/open-collective-fiscal-hosts">monthly webinars for Fiscal Hosts</a>, a space for you to learn and ask questions. <a href="https://events.humanitix.com/open-collective-fiscal-hosts">Sign up free</a>.
```
Comment (SudharakaP) [inline templates/emails/activated.collective.as.host.hbs:26]:
```suggestion
<p>I hold <a href="https://events.humanitix.com/open-collective-fiscal-hosts" target="_blank" rel="noopener noreferrer">monthly webinars for Fiscal Hosts</a>, a space for you to learn and ask questions. <a href="https://events.humanitix.com/open-collective-fiscal-hosts">Sign up free</a>.
```

File: templates/emails/activated.collective.as.host.hbs [hunk 2]
```diff
@@ -23,9 +23,11 @@ Contributions made to your Collectives via credit card (through Stripe) or or ot
...
 
-<p>We want to help your community thrive! Please feel free to reply to this email or <a href="http://calendly.com/piamancini/call">schedule a time to chat with me</a>.</p>
+<p>You're also welcome to <a href="https://slack.opencollective.com">join our Slack</a> and jump into the #hosts channel.
```
Comment (SudharakaP) [inline templates/emails/activated.collective.as.host.hbs:28]:
```suggestion
<p>You're also welcome to <a href="https://slack.opencollective.com" target="_blank" rel="noopener noreferrer">join our Slack</a> and jump into the #hosts channel.
```

File: templates/emails/activated.collective.as.host.hbs [hunk 3]
```diff
@@ -23,9 +23,11 @@ Contributions made to your Collectives via credit card (through Stripe) or or ot
...
+
+<p>We want to help your community thrive! Please feel free to reply to this email or <a href="http://calendly.com/piamancini/call">schedule a call with me</a>.</p>
```
Comment (SudharakaP) [inline templates/emails/activated.collective.as.host.hbs:30]:
```suggestion
<p>We want to help your community thrive! Please feel free to reply to this email or <a href="http://calendly.com/piamancini/call" target="_blank" rel="noopener noreferrer">schedule a call with me</a>.</p>
```

**Review by SudharakaP — CHANGES_REQUESTED** [review]
My thinking is ideally we want users to open a new tab in their browsers when opening these links. Therefore I suggest the following changes. 😄 

@Betree : Let me know if you think otherwise. 😄

**Comment by Betree** [comment]
> My thinking is ideally we want users to open a new tab in their browsers when opening these links. Therefore I suggest the following changes. smile
> 
> @Betree : Let me know if you think otherwise. smile

Adding that to emails does not seem necessary, most clients automatically open emails in new tabs. And even if some of them don't, I don't think it's our role to override links default behavior here, see https://uxdesign.cc/linking-to-a-new-tab-vs-same-tab-f88b495d2187:
> The only time it is recommended that you open a link in a new tab is when opening in the same screen would interrupt a process (e.g. when a user is filling out a form or viewing a video).

Most people know they can middle-mouse-click or right-click> "Open in a new tab" if they need to.

We currently don't do it in our other templates, so I would stick for consistency.

**Review by Betree — APPROVED** [review]

**Comment by SudharakaP** [comment]
@Betree : Sure thanks for the feed back; so let us stick to not adding those. 👍🏽

**Pull request merged**